lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -4926,15 +4947,15 @@ ActorIsolation ActorIsolationRequest::evaluate(
     return isolation;
   };
 
-  auto isolationFromAttr = getIsolationFromAttributes(value);
+  auto isolationFromAttr = getIsolationFromAttributes(value);      // add to interface request
```

Comment:
somewhat weird comments, remove the "floaty" comments or move them above the line of code it is about?

Reply:
oh no, looks like this comment slipped in from another branch , will remove it!